### DISCUSSION TITLE
Works with setup.cfg?

### DISCUSSION THREAD

**Discussion by Holt59** [ymyzk/tox-gh-actions#161] [Q&A] [answered]
I am trying to get this to work for one of my project where I am using `setup.cfg` for `tox` configuration instead of `tox.ini`.

My tox configuration looks like this:

```
[tox]
envlist = py35, py36, py37, py36-lint

[gh-actions]
python =
    3.5: py35
    3.6: py36, py36-lint
    3.7: py37

[testenv]
deps =
    pytest
commands =
    pytest tests

[testenv:py36-lint]
deps =
    black
    flake8
    flake8-black
    mypy
commands =
    black --check --diff setup.py minilp tests
    flake8 minilp tests
    mypy minilp tests
```

I have copy/pasted your workflow (simply removed python 3.5), but only the `py36-lint` environment is tested by `tox`, as if I ran `tox` without your `tox-gh-actions`... 

Any workaround?

My project is at https://github.com/Holt59/minilp/tree/packaging.

**Comment by ymyzk** [comment, marked as answer]
@Holt59 thanks for reporting the issue. It seems there's a misconfiguration in your `setup.cfg`.

Here's what I find out. I cloned your repository, checked out `packaging` branch and run `tox` command locally. Only `py36-lint` env was executed. Also, `tox -l` (show a list of testing) failed.

I checked the tox's documentation and found

> Note that setup.cfg requires the content to be under the tox:tox section. pyproject.toml on the other hand is in TOML format.

I made the following change and it seems to have fixed the problem.

```diff --git a/setup.cfg b/setup.cfg
index 6d4a94a..ea80b52 100644
--- a/setup.cfg
+++ b/setup.cfg
@@ -21,7 +21,7 @@ ignore_missing_imports = True
 [mypy-pytest.*]
 ignore_missing_imports = True
 
-[tox]
+[tox:tox]
 envlist = py35,py36,py37,py36-lint
 
 [gh-actions]
```

**Comment by Holt59** [comment]
@ymyzk Thanks for the information. After updating my `setup.cfg`, everything works as expected!
